test/cases/ostree-simplified-installer.sh

```diff
@@ -495,9 +495,15 @@ sudo virt-install --name="${IMAGE_KEY}-http"\
                   --boot uefi,loader_ro=yes,loader_type=pflash,nvram_template=/usr/share/edk2/ovmf/OVMF_VARS.fd,loader_secure=no \
                   --nographics \
                   --noautoconsole \
-                  --wait=-1 \
+                  --wait=15 \
                   --noreboot
 
+# Installation can get stuck, destroying VM helps
+# See https://github.com/osbuild/osbuild-composer/issues/2413
+if [[ $(sudo virsh domstate "${IMAGE_KEY}-http") == "running" ]]; then
+    sudo virsh destroy "${IMAGE_KEY}-http"
+fi
+
```

Comment:
This may be a stupid question, but should not this conditional and the `virt-install` command from above be in some kind of a for loop? I expect that the VM still needs to be installed for the test to pass, so just destroying it may unstuck the test, but it also means that the VM was not installed successfully, right?

Reply:
I don't think there is a loop happening, IMO it's just finished and waiting for something and AFIAK nobody knows what, am I right @henrywang ? 

So what this does is that if after 15 minutes of running `virt-install` it's expected that it's finished and the condition is there only to prevent failure in case the `virt-install` was successful and the VM was turned off already. Of course this doesn't fix anything but it's better than disabling the test entirely. And in case it was destroyed prematurely the following ansible check will fail or the VM won't boot at all so it won't introduce any false positives.

Reply:
Putting it into a loop would be probably necessary in case the stuck installation meant failed installation which apparently is not the case here.

Reply:
OK, thanks for the explanation. It the VM is actually stuck after successful installation, then it makes sense to do what you did in the PR. It makes sense now 🙂